frontend/src/api/createGame.ts

```diff
@@ -0,0 +1,17 @@
+import axios, { AxiosResponse } from "axios";
+import ApiResponse from "./ApiResponse";
+import axiosCall from "./axiosCall";
+
+// Expected status codes
+const CREATED_201 = 201;
+// const SERVER_ERROR_500 = 500;
```

Comment:
Here... 👀

Reply:
Yeah wasn't sure whether to keep these or not. They won't be used in these files but they could be nice documentation to show what the expected error codes are - though this could just be explained in a regular comment